app/src/main/res/layout/activity_main.xml

```diff
@@ -9,19 +9,16 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recycler_view"
-        android:layout_width="403dp"
+        android:layout_width="0dp"
         android:layout_height="518dp"
         android:layout_marginStart="1dp"
         android:layout_marginLeft="1dp"
-        android:layout_marginTop="1dp"
         android:layout_marginEnd="1dp"
         android:layout_marginRight="1dp"
         app:layoutManager="LinearLayoutManager"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="1.0" />
+        app:layout_constraintStart_toStartOf="parent" />
```

Comment:
Jeśli wciąż masz problem z położeniem przycisku, to sugeruję zrobić tak:
* dla `new_user_button` usunąć `app:layout_constraintVertical_bias`
* dla `recycler_view` dodać `app:layout_constraintTop_toTopOf="@+id/new_user_button"` oraz `android:layout_height="0dp"` - w ten sposób recyclerView rozciągnie się od buttona do dołu ekranu

Reply:
Nic nie zrobiłam a przycisk wrócił na swoje miejsce 😄 
Eksperymentowałam z parametrami i po usunięciu `app:layout_constraintVertical_bias` nie było żadnej zmiany więc usunęłam.